curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/62ac9e72adf8e20ec1008da0.md

```diff
@@ -0,0 +1,120 @@
+---
+id: 62ac9e72adf8e20ec1008da0
+title: Step 42
+challengeType: 0
+dashedName: step-42
+---
+
+# --description--
+
+The last border looks a little out of place. You can select the last
+element of a specific type using the `last-of-type` CSS pseudo-class, like this: 
+
+```
+p:last-of-type { }
+```
+
+That will select the last `p` element. Create a new selector that targets the last `fieldset` element and set its `border-bottom` to `none`.
+
+# --hints--
+
+You can use the `:last-of-type` pseudo-class to select only the last element of a certain type.
+
+```js
+assert.exists(new __helpers.CSSHelp(document).getStyle('fieldset:last-of-type'));
+```
+
```

Comment:
Missing test text

Reply:
I can't emote because I am using the phone, so here👍. Think back do I actually need to emote.🤔